monodoc/monodoc.csproj

```diff
@@ -1,713 +1,27 @@
-      <LogicalName>protenumeration.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protevent.png">
-      <LogicalName>protevent.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protextension.png">
-      <LogicalName>protextension.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protfield.png">
-      <LogicalName>protfield.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protinterface.png">
-      <LogicalName>protinterface.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protmethod.png">
-      <LogicalName>protmethod.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protproperty.png">
-      <LogicalName>protproperty.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/protstructure.png">
-      <LogicalName>protstructure.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubclass.png">
-      <LogicalName>pubclass.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubdelegate.png">
-      <LogicalName>pubdelegate.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubenumeration.png">
-      <LogicalName>pubenumeration.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubevent.png">
-      <LogicalName>pubevent.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubextension.png">
-      <LogicalName>pubextension.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubfield.png">
-      <LogicalName>pubfield.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubinterface.png">
-      <LogicalName>pubinterface.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubmethod.png">
-      <LogicalName>pubmethod.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubproperty.png">
-      <LogicalName>pubproperty.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/pubstructure.png">
-      <LogicalName>pubstructure.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/reference.png">
-      <LogicalName>reference.png</LogicalName>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Resources/images/treebg.png">
-      <LogicalName>treebg.png</LogicalName>
-    </EmbeddedResource>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="monodoc.dll.config">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <PackageReference Include="SharpZipLib" Version="1.0.0-alpha2" />

```

Comment:
One of the nice things about PackageReference is that the Pack target automatically converts it into a NuGet package dependency.